### PR TITLE
[CAN MERGE FIRST] Fix bugs

### LINKED ISSUE #218: [PE-D] Inputting `edit` does not return any error
## Summary
Merely typing `edit` does not return any error messages, unlike keying in `add` which displays the following. This can be confusing for the user who can think that the program is not processing the command.

Error message from keying in just `add` in the command input box:
> Invalid command format! 
add: Adds a player, lineup, schedule to MyGM
add: Adds a player to MyGM. 
Parameters: P/ n/NAME p/PHONE e/EMAIL h/HEIGHT j/JERSEYNUMBER w/WEIGHT [t/TAG]...
Example: add P/ n/John Doe p/98765432 e/johnd@example.com h/180 j/23 w/80 t/PG t/SG
add: Adds a lineup to MyGM.
Parameters: L/ n/LINEUP NAME[P/PLAYER]...
Example: add L/ n/Starting 5
add: Adds a schedule to MyGM.
Parameters: S/ n/SCHEDULE NAME r/SCHEDULE DESCRIPTION d/DATE TIME
Example: add S/ n/finals r/nba finals d/01/01/2022 2000

## Screenshot

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: sbhbenjamin/ped#8

### LINKED ISSUE #226: [PE-D] Runtime Error when user inputs edit command
According to the documentation, "edit P/Alex Yeoh T/SF" should be a valid command.

However, when I type "edit P/Alex Yeoh T/SF" on the software, attempting to edit the tag of Alex Yeoh, the system crashes and the user input does not disappear upon pressing enter. I checked my command prompt and realized that there is runtime error, as shown above.

This is a severe bug as the system should not crash given a valid input, even if input is invalid, the system also should not crash.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: bingsen0806/ped#10

### LINKED ISSUE #228: [PE-D] Severe data inconsistency issue after editing
How to reproduce the issue:

Step 1: Create a new lineup (e.g. test lineup) and add any player (e.g. Alex Yeoh) to the lineup

Step 2: Change the name of Alex Yeoh (e.g. to New Name)

Step 3: Change the name of "test lineup" (e.g. to "new lineup")

Step 4: "view L/" to view all lineups

The following is shown:

Firstly, "Alex Yeoh" should not be shown in system response, since the player "Alex Yeoh" is now renamed as "New Name", so "New Name" should be shown instead.

Secondly, neither "New Name" nor "Alex Yeoh" is shown in the list of players (only Greek Koumpo  etc is shown). This is incorrect because "New Name" (or the old "Alex Yeoh) should still be in lineup named "new lineup" (or the old "test lineup").

This is an data inconsistency issue that affects the correctness of the whole software, and is the most severe bug out of all other bugs reported.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: bingsen0806/ped#12

### LINKED ISSUE #235: [PE-D] Inconsistency between documentation and application for editing a player's tags
## Summary
The documentation mentions that editing a player can be done with the syntax `[T/TAGS]`, however, when this is input in the application, it is invalid. Is this a typo where it is supposed to be `[t/TAG]`?

## Screenshot
Documentation:

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: sbhbenjamin/ped#15

### LINKED ISSUE #237: [PE-D] Wrong command format for Edit command in User Guide
The part circles in red should be "[t/TAG]..." . T/ does not work, and should be t/ (already reported in another issue about crashing and having runtime error, but this is a separate report for documentation)

It should not be shown as t/TAGS because the user will be mislead to input t/PG SF SG, when in fact the correct way is t/PG t/SF t/SG

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: bingsen0806/ped#14

### PR DESCRIPTION
**Pull request by FYimu**
- [x] solve data inconsistency #228
- [x] solve runtime error #226 
- [x] catch error for `edit` #218 
- [ ] editted tags **cannot replicate bug** #206

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ Update the details of a player, lineup or schedule.
 
 **To edit a player:**
 
-Format: `edit P/NAME [n/NAME] [p/PHONE_NUMBER] [w/WEIGHT] [h/HEIGHT] [j/JERSEY_NUMBER] [T/TAGS]`
+Format: `edit P/NAME [n/NAME] [p/PHONE_NUMBER] [w/WEIGHT] [h/HEIGHT] [j/JERSEY_NUMBER] [t/TAG]…​`
 
 * Edit the details of a player from the player list.
```
Comment (DALIN-Prog) [inline docs/UserGuide.md:270]: Thanks for the amendments! Was thinking if you want to also mention that removal of tags can/ must be done in edit command

Edit:
Ignore this, I forgotten that I have assigned myself for this task😅

Comment (FYimu) [reply docs/UserGuide.md:270]: HAHA okies

**Comment by DALIN-Prog** [comment]
Looks clean, thanks!

Can I also request that you drop a comment in issues #206 that bug cannot be replicated?

**Review by DALIN-Prog — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -265,7 +265,7 @@ Update the details of a player, lineup or schedule.
 
 **To edit a player:**
 
-Format: `edit P/NAME [n/NAME] [p/PHONE_NUMBER] [w/WEIGHT] [h/HEIGHT] [j/JERSEY_NUMBER] [T/TAGS]`
+Format: `edit P/NAME [n/NAME] [p/PHONE_NUMBER] [w/WEIGHT] [h/HEIGHT] [j/JERSEY_NUMBER] [t/TAG]…​`
```
Comment (DALIN-Prog) [inline docs/UserGuide.md:268]: closes #237, #235

**Review by DALIN-Prog — APPROVED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (FYimu) [reply docs/UserGuide.md:268]: Thanks for helping to link all the issues!!! (I just realized how to do it correctly just now 😅

**Pull request merged**